<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDbDistributorComponent</class>
 <widget class="QFrame" name="CDbDistributorComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search and find distributors</string>
  </property>
  <property name="whatsThis">
   <string>Search and find distributors</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="vl_DistributorComponent">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="fr_TopBar">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="hl_TopBar">
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <widget class="swift::gui::filters::CDistributorFilterBar" name="filter_Distributor">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hs_TopBar">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="wi_SimulatorSpecific" native="true">
        <layout class="QHBoxLayout" name="hl_FSXSpecific">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="pb_SelectAllFsFamily">
           <property name="text">
            <string>FS standard</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_SelectXPlaneBB">
           <property name="text">
            <string>XPlane BlueBell</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_SelectXplaneXCSL">
           <property name="text">
            <string>XPlane XCSL</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="swift::gui::views::CDistributorView" name="tvp_Distributors">
     <property name="dragDropMode">
      <enum>QAbstractItemView::DragOnly</enum>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>swift::gui::views::CDistributorView</class>
   <extends>QTableView</extends>
   <header>gui/views/distributorview.h</header>
  </customwidget>
  <customwidget>
   <class>swift::gui::filters::CDistributorFilterBar</class>
   <extends>QFrame</extends>
   <header>gui/filters/distributorfilterbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pb_SelectAllFsFamily</tabstop>
  <tabstop>pb_SelectXPlaneBB</tabstop>
  <tabstop>tvp_Distributors</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
